scripts/skills/perks/perk_rf_calculated_strikes.nut

```diff
@@ -0,0 +1,51 @@
+this.perk_rf_calculated_strikes <- this.inherit("scripts/skills/skill", {
+	m = {
+		DamageMult = 1.2,
+		InitiativeMult = 0.8
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_calculated_strikes";
+		this.m.Name = ::Const.Strings.PerkName.RF_CalculatedStrikes;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_CalculatedStrikes;
+		// this.m.Icon = "ui/perks/rf_calculated_strikes.png";	// TODO: add this icon
+		this.m.Type = ::Const.SkillType.Perk;
+		this.m.Order = ::Const.SkillOrder.Perk;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = false;
+	}
+
+	function onUpdate( _properties )
+	{
+		_properties.InitiativeMult *= this.m.InitiativeMult;
+	}
+
+	function onAnySkillUsed( _skill, _targetEntity, _properties )
+	{
+		if (_skill != this) return;
```

Comment:
This is wrong. The `_skill` is the skill being used e.g. `Slash`. The `isEnabledFor` check you have is the only one you need.